assets/queries/k8s/container_security_context/query.rego

```diff
@@ -0,0 +1,52 @@
+package Cx
+
+CxPolicy [ result ] {
+    document := input.document[i]
+    specInfo := getSpecInfo(document)
+    
+    spec := document.spec
+    
+    document.kind == "Pod"
+    not spec.securityContext
+
+	result := {
+                "documentId": 		  input.document[i].id,
+                "issueType":		  "MissingAttribute",
+                "searchKey": 	      sprintf("%s", [specInfo.path]),
+                "keyExpectedValue":   sprintf("Pod has a security context applied in document[%d]",[i]),
+                "keyActualValue": 	  sprintf("Pod has not a security context applied in document[%d]",[i])
+              }
+}
+
+
+CxPolicy [ result ] {
+    document := input.document[i]
```

Comment:
Add metadata information to get a more specific searchKey
metadata := document.metadata